File: saizad-mvvm/src/main/java/com/saizad/mvvm/MultiDexApplication.java
package com.saizad.mvvm;

import android.content.Context;
import androidx.annotation.CallSuper;
import androidx.multidex.MultiDex;
import dagger.android.DaggerApplication;

public abstract class MultiDexApplication extends DaggerApplication {

    @Override
    @CallSuper
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(newBase);
        MultiDex.install(this);
    }

}

File: saizad-mvvm/src/main/java/com/saizad/mvvm/MultiDexApplication.kt
package com.saizad.mvvm

import android.app.Application
import android.content.Context
import androidx.annotation.CallSuper
import androidx.multidex.MultiDex

open class MultiDexApplication : Application() {

    @CallSuper
    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(newBase)
        MultiDex.install(this)
    }

}